internal/kokoro/vet.sh

```diff
@@ -25,6 +25,9 @@ find . -type f -name "*.go" ! -name "*.pb.go" -exec grep -L "\(Copyright [0-9]\{
 # Fail if a dependency was added without the necessary go.mod/go.sum change
 # being part of the commit.
 go mod tidy
+pushd v2
+  go mod tidy
+popd
```

Comment:
Should v2 get `git diff` as well?

Reply:
git diff will look for any changes that had happened in the v1 go mod tidy as well as the v2 go mod tidy. That is: git diff looks at the whole repo, so there's no distinction between the two dirs :)